client/src/main/java/org/apache/cassandra/sidecar/client/SidecarClientConfigImpl.java

```diff
@@ -68,19 +76,39 @@ public long maxRetryDelayMillis()
         return maxRetryDelayMillis;
     }
 
+    /**
+     * @return the minimum amount of time to wait before retrying a failed health check
+     */
+     @Override
+     public Duration minimumHealthRetryDelay()
+     {
+        return minimumHealthRetryDelay;
+     }
+
+    /**
+     * @return the maximum amount of time to wait before retrying a failed health check
+     */
+     @Override
+     public Duration maximumHealthRetryDelay()
+     {
+        return maximumHealthRetryDelay;
+     }
+
     public static Builder builder()
     {
         return new Builder();
     }
 
     /**
-     * {@code SidecarConfig} builder static inner class.
+     * {@code SidecarConfig} builder static inner class
```

Comment:
removing the period is unnecessary.

Reply:
Added periods back here and in several other places.